can/map/test-real-time-super-model.js

```diff
@@ -152,7 +154,7 @@ module.exports = function(makeTypes){
 
 
     	function serverSideDuplicateCreate(){
-    		connection.createInstance({id: 10, due: "today",createdId: 1, type: "important"}).then(function(createdInstance){
```

Comment:
`createdId` should be `createId`. This just happened to pass since createInstance is getting the instance from the instance store anyway.